### DISCUSSION TITLE
request a client version without gui

### DISCUSSION THREAD

**Discussion by 023010** [rustdesk/rustdesk#6209] [General]
is there any plan to develop a client which doesnot provide remote support and gui (configure it with command line and run in background)? it just sets up unattended access to a device similar with teamviewer host (used in my tvbox or server at home).

**Comment by csigristgit** [comment]
+1

**Comment by xlionjuan** [comment]
Go use Tailscale or Cloudflare ZeroTrust browser SSH

**Comment by rustdesk** [comment]
No plan
